proto/data.proto

```diff
@@ -118,7 +118,8 @@ message SplitTrigger {
 // for the merge to be completed and put into operation.
 message MergeTrigger {
   optional RangeDescriptor updated_desc = 1 [(gogoproto.nullable) = false];
-  optional int64 subsumed_raft_id = 2 [(gogoproto.nullable) = false, (gogoproto.customname) = "SubsumedRaftID"];
+  optional int64 subsumed_raft_id = 2 [(gogoproto.nullable) = false,
+    (gogoproto.customname) = "SubsumedRaftID", (gogoproto.casttype) = "RaftID"];
```

Comment:
Some of these lines look to have different indentation than others. Compare this line to lines 100 and 102.